### PR TITLE
Fix 👌: Use sh -c to ensure that the piped file has access to write in…

### PR DESCRIPTION
**Pull request by ivaruf**
One line, read the commend :D

### CODE REVIEW

File: cmd/okctl/completion.go [hunk 1]
```diff
@@ -18,7 +18,7 @@ Bash:
 
 	# To load completions for each session, execute once:
 	# Linux:
-	$ okctl completion bash > /etc/bash_completion.d/okctl
+	$ sudo sh -c 'okctl completion bash > /etc/bash_completion.d/okctl
```
Comment (yngvark) [inline cmd/okctl/completion.go:21]: Add `'` in the end

**Review by yngvark — CHANGES_REQUESTED** [review]

File: cmd/okctl/completion.go (hunk 1 above)
Comment (ivaruf) [reply cmd/okctl/completion.go:21]: ... damn it, I had one job

Comment (ivaruf) [reply cmd/okctl/completion.go]: Fixed

**Review by yngvark — APPROVED** [review]
:tada:

**Pull request merged**